### PR TITLE
Release 5.0.3

### LINKED ISSUE #247: The filename and paths are different (worse) in the latest update
In 5.0.0 it was (expected):
> /myapp/cypress/snapshots/base/test.cy.js/forgot-password-android.png

as I call command as
```
cy.compareSnapshot('forgot-password-android')
```

Now in 5.0.2:
> /myapp/cypress/snapshots/base/cypress/e2e/test.cy.js/forgot_password_android.png

Extra `/cypress/e2e` path and the file name for some reason converted to snakecase.

### PR DESCRIPTION
**Pull request by Zaista**
This PR will:

- remove the sanitation from `commands.ts` file, add generic sanitize to `updateSnapshot` function. Fixes #247 and #252
- add tests to check if the file name pattern is appropriate
- add a forgotten test that would catch the previous bug fix for #253

### CODE REVIEW

File: src/command.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ function addCompareSnapshotCommand(screenshotOptions?: ScreenshotOptions): void
         }
       }
 
-      const nameSanitized = name.replace(/[^a-z0-9]/gi, '_').toLowerCase()
+      const nameSanitized = name.replace(/[<>:"/\\|?*]/g, '_')
```
Comment (area73) [inline src/command.ts:79]: You are not scaping space " ". Is that intentional?
I'm wondering how it will eventually work on linux / windows servers.

Comment (area73) [reply src/command.ts:79]: maybe we can also do a test for this use case to be safe

Comment (Zaista) [reply src/command.ts]: In the meantime, I have actually completely removed the custom sanitation from the frontend and only rely on sanitize from the [sanitize-filename](https://www.npmjs.com/package/sanitize-filename) dependency. In the meantime I'll add the space character tests and handle it if needed

Comment (Zaista) [reply src/command.ts]: seems to be working without issues

Comment (Zaista) [reply src/command.ts]: I specifically added a 'file names' tests to run on all platforms, and they all passed

**Comment by Zaista** [comment]
Ready for review

**Review by area73 — APPROVED** [review]

**Pull request merged**